canonical/charmhub.io
# Add category to interfaces

## Done
- Add category to interfaces
## How to QA
- Go to https://charmhub-io-1512.demos.haus/interfaces.json and check that each interface has a category
## Issue / Card
Fixes #

## Screenshots
